src/main/java/jp/ikedam/jenkins/plugins/scoringloadbalancer/ScoringLoadBalancer.java

```diff
@@ -337,6 +392,8 @@ public boolean configure(StaplerRequest req, JSONObject json) throws hudson.mode
             // reset optional authentication to default before data-binding
             this.enabled = true;
             this.reportScoresEnabled = false;
+            this.simultaneousBuildsWorkaroundEnabled = false;
+            this.simultaneousBuildsWorkaroundThrottleTime = 1000;
```

Comment:
Can we cover this by tests?

Reply:
It wasn't before and it's a deprecated method, however I now created a test for it: [`5074ba0` (#54)](https://github.com/jenkinsci/scoring-load-balancer-plugin/pull/54/commits/5074ba0c785a9cb1aa7e5958849640aa2f255a9d)